pkg/flow/adapter/xmltojsontransformation/adapter.go

```diff
@@ -46,6 +47,10 @@ type envAccessor struct {
 	BridgeIdentifier string `envconfig:"EVENTS_BRIDGE_IDENTIFIER"`
 	// CloudEvents responses parametrization
 	CloudEventPayloadPolicy string `envconfig:"EVENTS_PAYLOAD_POLICY" default:"error"`
+	// Sink defines the target sink for the events. If no Sink is defined the
+	// events are replied back to the sender.
+	// +optional
+	Sink *string `envconfig:"K_SINK"`
```

Comment:
Should make use of the `omitempty` annotation

Reply:
This is an envConfig struct, it doesn't get marshaled to JSON :)